android/build.gradle

```diff
@@ -15,12 +15,12 @@ apply plugin: 'com.android.library'
 apply plugin: 'bintray-release'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.2"
+    compileSdkVersion androidCompileSdkVersion
+    buildToolsVersion androidBuildToolsVersion
 
     defaultConfig {
-        minSdkVersion 14
-        targetSdkVersion 23
+        minSdkVersion androidMinSdkVersion
+        targetSdkVersion androidTargetSdkVersion
```

Comment:
Yep, we shouldn't. Here is the change done in Dagger by @JakeWharton in https://github.com/google/dagger/pull/591

When you specify this happens:
- if it is less then the client app's target, no affect at all
- If it is greater, then it takes affect and change the targetSdk of the client.

In either cases, it is not good.